flow-typed/types.js

```diff
@@ -296,35 +296,40 @@ declare type NetworksData = {
 /**
  * Digital assets
  */
-declare type DigitalAssetsListType = 'balance' | 'popular' | 'custom'
+declare type DigitalAssetsFilter = {|
+  +hideZeroBalance: boolean,
+  +myAssetsFirst: boolean,
+  +sortByName: false | 'asc' | 'desc',
+  +sortByBalace: false | 'asc' | 'desc',
+|}
+
+declare type DigitalAssetBalances = {
+  [string]: {
+    balance: Bignumber,
+    lastUpdated: ?Date,
```

Comment:
I suggest `updatedAt`, like `createdAt` (e.g. default field name for several DBs, etc)